content/blog/2021-06-16-june-21-dvc-heartbeat.md

```diff
@@ -0,0 +1,242 @@
+﻿---
+title: June '21 Heartbeat
+date: 2021-06-16
+description: |
+  Monthly updates are here! Making sense of the MLOps Landscape, MLOps 
+  Community love, our MLOps philosophy, DVC Studio, R for DVC, new learning
```

Comment:
MLOps a bit repetitive plus sounds like we could merge these 2 from the content?

```suggestion
  Monthly updates are here! Making sense of the MLOps Landscape and
  Community, our MLOps philosophy, DVC Studio, R for DVC, new learning
```

Reply:
not making sense of the Community, so just dropped second MLOps so the just "Community love,..."

Reply:
OK sure, up to you. I just thought it was repetitive but. Not a strong opinion